Dockerfile

```
@@ -0,0 +1,30 @@
+FROM ubuntu
+
+LABEL version="0.1"
+LABEL description="A super docker image."
+
+RUN apt-get update
+
+RUN apt-get -y install curl git gnupg
+RUN curl -sL https://deb.nodesource.com/setup_12.x -O
+RUN bash setup_12.x
+
+RUN apt-get -y install nodejs
+
+RUN mkdir -p /home/app 
+
+WORKDIR /home/app
+
+ADD server.js server.js
+ADD index.html index.html
+ADD package.json package.json
+
+RUN apt-get install -y wget
+RUN wget https://www.lescinskas.lt/assets/img/posts/2019/docker.png -O background.png
```

The `ADD` instruction has multiple usages like retrieving remote artifacts for GitHub but should be avoided for simple copy, you should use the `COPY` instruction in this case.

In my suggestion, I copy all the files using the `COPY` instruction only one time to reduce the number of layer. I also removed the `wget` package in favor of the `ADD` instruction which is able to retrieve remote files.

You can find more on the `ADD vs COPY` [here](https://docs.docker.com/build/building/best-practices/#add-or-copy).

```suggestion
COPY server.js index.html package.json ./

ADD https://www.lescinskas.lt/assets/img/posts/2019/docker.png ./
```

On another note, directly including assets like images(png, webp, ...) should be reconsidered in favor of a dedicated assets managements. You can read about it on [IBM](https://www.ibm.com/think/topics/digital-asset-management#:~:text=Digital%20asset%20management%20(DAM)%20can,can%20access%20their%20media%20assets.) for example.